tests/utils/test_utils.py

```diff
@@ -0,0 +1,30 @@
+# Copyright (C) 2019-2024 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+from typing import Any, List
+
```

Comment:
Consider splitting and moving the `datumaro/util/test_utils.py` file as well.

Reply:
tried to do it, and it causes changes in 80 files. Will do it in another PR in order not to make this PR even bigger

Reply:
made a PR with it: https://github.com/cvat-ai/datumaro/pull/52